app/views/mailers/notify_mailer/new_badge_email.html.erb

```diff
@@ -26,7 +26,7 @@
     </a>
   </p>
 
-  <% unless @badge.credits_awarded.zero? %>
+  <% if @badge.credits_awarded > 0 && Listing.feature_enabled? %>
```

Comment:
comment, nonblocking:  should the badge know how to decide this? Something like `@badge.award_credits?` (since the badge also knows how many credits to award). 

I see us adding conditional logic in the views, and imagine it is a stopgap.

Reply:
Definitely a stop gap…and like the idea of `@badge.award_credits?` but think the stronger solution is asking the badge for what credit text it should render.  The challenge being that credits are only useful with listings but that may not be the case in the future.